CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [PEP 440](https://www.python.org/dev/peps/pep-0440/) 
 and uses [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.13.0]
+### Added
+- Optional ability to specify the name of the dockerfile to build with the `reusable-docker-ghcr` action.
```

Comment:
12.0 is unreleased, by the way. So this is a second new minor release

Reply:
Noted, we will fix that before releasing.